angular.json

```diff
@@ -35,8 +35,13 @@
                         "scripts": [
                             "node_modules/marked/lib/marked.js"
                         ],
-                        "es5BrowserSupport": true,
-                        "showCircularDependencies": false
+                        "showCircularDependencies": false,
+                        "allowedCommonJsDependencies": [
+                          "moment",
```

Comment:
Isn't it missing angular CDK?

Reply:
CDK is treated as default angular library. It's not required to be added.